lib/rules/no-duplicate-at-import-rules/index.js

```diff
@@ -73,6 +70,52 @@ const rule = (primary) => {
 	};
 };
 
+/**
+ * List the import conditions found in the prelude of an `@import` rule
+ *
+ * @param {Node[]} params
+ * @typedef {import('postcss-value-parser').Node} Node
+ * @returns {Array<string>}
+ */
+function listImportConditions(params) {
+	if (!params.length) return [];
+
+	/** @type {Array<Array<Node>>} */
```

Comment:
[suggest] Maybe we can simplify the `media` type:

```suggestion
	/** @type {Array<Node>} */
```

With `.push(...array)` pattern:

```js
if (param.type === 'div' && param.value === ',') {
	media.push(...lastMediaQuery);
	lastMediaQuery = [];
	continue;
}
```

```js
media.push(...lastMediaQuery);
```

Reply:
That would not be correct.

This function should split media query lists on comma's.
If we use the spread operator `const media = []` would be an array of parts of media queries, not an array of media queries.

`@media (color: 1) and (hover) {}`

`['(color:1)', 'and', '(hover)']`

Reply:
I see. Let's leave it. 👍🏼

Reply:
I went over this function one more time checking this and found that serializing a bit earlier is also a way to simplify the types.

In doing so I also found a bug in the removal of whitespace.

Reply:
Thanks! Easier to read now 👍🏼